var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -282,7 +285,7 @@ class Mfem(Package, CudaPackage, ROCmPackage):
 
     depends_on("mpi", when="+mpi")
     depends_on("hipsparse", when="@4.4.0:+rocm")
-    depends_on("hipblas", when="@4.4.0:+rocm")
+    depends_on("hipblas", when="@4.8.0:+rocm")
```

Comment:
Why are we moving the lowerbound here? Does mfem < 4.8.0 not support hipblas?

Reply:
Yes, before v4.8, MFEM does not depend (directly) on `hipblas`.

Reply:
Gotcha, so this was an error in the Spack package prior?

Reply:
Yes.